inputFiles/poromechanics/PoroElastic_staircase_singlephase_3d_seq.xml

```diff
@@ -0,0 +1,344 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Solvers>
+
+    <SinglePhaseReservoirPoromechanics
+      name="reservoirPoromechanics"
+      solidSolverName="linearElasticity"
+      reservoirAndWellsSolverName="reservoirSystem"
+      logLevel="1"
+      targetRegions="{ channel, barrier, wellRegion1, wellRegion2 }">
+      <NonlinearSolverParameters
+        couplingType="Sequential"
```

Comment:
I know you are just testing right now, but we should eventually use the xml include functionality to have a `PoroElastic_staircase_singlephase_3d_base.xml` and then two concise downstream files, `_fim` and `_seq`.  We are only really changing two lines.

Reply:
done